src/app/earn/constants.ts

```diff
@@ -0,0 +1,51 @@
+import { ARBITRUM, BASE, MAINNET } from '@/constants/chains'
+import {
+  DiversifiedStakedETHIndex,
+  ETH,
+  HighYieldETHIndex,
+  icETHIndex,
+  Token,
+  USDC,
+  USDT,
+  WBTC,
+  WETH,
+} from '@/constants/tokens'
+import { getAddressForToken } from '@/lib/utils/tokens'
+
+export const yieldTokens = [
+  HighYieldETHIndex,
+  icETHIndex,
+  DiversifiedStakedETHIndex,
+]
+
+export function getCurrencyTokens(chainId: number): Token[] {
+  switch (chainId) {
+    case MAINNET.chainId:
+    case ARBITRUM.chainId:
+      return [ETH, WETH, WBTC, USDC, USDT]
+    case BASE.chainId:
+      return [ETH, WETH, USDC]
+    default:
+      return []
+  }
+}
+
+export function getYieldTokens(chainId: number): Token[] {
```

Comment:
Q: same as above

Reply:
Slightly different issue here - CDETI is being returned and it's not a yield token. Also marked it as a TODO - will fix when we update the tokenlists.